pkg/base1/cockpit.js

```diff
@@ -926,6 +926,29 @@ function basic_scope(cockpit) {
         });
     };
 
+    /* Not public API ... yet? */
+    cockpit.expect_restart = function expect_restart(delay, host) {
```

Comment:
Do you think that we have an expect API? For various global events? And then have 'restart' as a code?